app/dashboards/user_dashboard.rb

```diff
@@ -11,7 +11,7 @@ class UserDashboard < Administrate::BaseDashboard
     id: Field::Number,
     email: Field::String,
     role: EnumField,
-    password: Field::String,
+    password: Field::String.with_options(searchable: false),
```

Comment:
Instead of specifying searchable false here, lets take advantage of [Administrate's `Field::Password` type](https://github.com/thoughtbot/administrate/blob/master/lib/administrate/field/password.rb) which will do the searchable false configuration by default and be more descriptive of the field attribute type.

```suggestion
    password: Field::Password,
```

Reply:
Hi @lauramosher !
Yeah, that looks better, I will take it!